trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.strip[0...10]
```

Comment:
[OPTIONAL] I think it would be better if you used a conditional statement to check if the name is actually up to 10. This would save us the process of running the strip method unnecessarily
[REQUIRED] Since Ruby uses  0 indexing, I think your range should stop at `9`.